### ISSUE TITLE
fps drop on every browser

### ISSUE THREAD

**Issue by Renalax** [haxball/haxball-issues#1995]
Hi, i cant literally play haxball cause if i unlock the fps with the command "--disable-frame-rate-limit" and the other one with "angle something"   browser will start lag in 5-10 mins of playing on every browser, and i feel so slow with high extra. How can i fix that? i played with like 1000 fps or something and still playing with same fps or higher sometimes but it freeze sometimes, is not my pc cause i can play everygame and get 0 lag my net is ok too. How can i fix that?

**Comment by Always-banned** [comment]
I don't know if we have the same problem ... but I also just started having this problem 3 days ago, I've tried different browsers but it's the same situation on all of them.

The game has become blurry and starts to freeze ... at first I thought it was me, but I see that more people have this problem.

I have been playing the game exclusively through the browser for years.
When the fps is locked at 144, the game is fine, that is. "smooth", but then I don't use full potential

But when I unlock FPS the game starts to be "fuzzy" and the image freezes every few seconds and my CPU usage starts to be 90%

Edit:
Games that use more GPU or any other power like CSGO/LoL etc.. are doing same just like before... so I wonder if something happend in HaxBall or in all browsers generally :/

**Comment by Renalax** [comment]
yeah i have the same problem. @basro help us plss :(

**Comment by basro** [comment]
@Renalax does the game become blurry for you too?

**Comment by Renalax** [comment]
@basro yeah starting feeling slow, high cpu and then after some minutes it will freeze and lag like 2 fps

**Comment by basro** [comment]
I'm asking if the image becomes blurry.

**Comment by Renalax** [comment]
yes

**Comment by GLWP** [comment]
Do you use the 'Use low-latency canvas' option? My screen blurs when I use this option. It feels like turning on the anti-aliasing option in a 3D game.

**Comment by Arxevah** [comment]
@basro cannot play the game when fps limit is unlocked. Tried literally every browser on the net possibly.

**Comment by Arxevah** [comment]
![resim_2023-06-29_004130645](https://github.com/haxball/haxball-issues/assets/110306866/7bb2802b-d953-4e73-94d7-355b2a43506b)
@basro

**Comment by Renalax** [comment]
same i had to quit the game and stop playing competitive, also if u use the angle option after diasble frame limit is working but still slow.

**Comment by Arxevah** [comment]
Uploading haxball fps.mp4…

**Comment by Suquer007** [comment]
"I think we have the same problem, or a similar one."

**Comment by wilmoot** [comment]
for me when canvas is active there is like 1 second of input lag and without it just frame drops when I have high fps

**Comment by iamtunayildiz** [comment]
If you clear your Google history, the problem is solved.

**Comment by iamtunayildiz** [comment]
> Google geçmişinizi temizlerseniz sorun çözülür.

This method works for me.
